playground/frontend/lib/components/link_button.dart

```diff
@@ -20,41 +20,62 @@ import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/svg.dart';
 import 'package:playground/src/assets/assets.gen.dart';
+import 'package:playground_components/playground_components.dart';
 import 'package:url_launcher/url_launcher.dart';
 
+const datasetsFolderPath =
+    'https://github.com/apache/beam/blob/master/playground/backend/datasets/';
+
 class LinkButton extends StatelessWidget {
   final String iconPath;
   final String text;
-  final String url;
+  final Iterable<String> urls;
```

Comment:
Revert to a single URL.
Opening multiple tabs in a single click is not user-friendly, and many browsers simply block multiple tabs.
By the time we support multiple datasets in an example we will make a more advanced widget to view them.

Reply:
done